apps/web/src/components/Menu/GlobalSettings/SettingsModal.tsx

```diff
@@ -178,26 +179,28 @@ const SettingsModal: React.FC<React.PropsWithChildren<InjectedModalProps>> = ({
                   }}
                 />
               </Flex>
-              <Flex justifyContent="space-between" alignItems="center" mb="24px">
-                <Flex alignItems="center">
-                  <Text>{t('Allow notifications')}</Text>
-                  <QuestionHelper
-                    text={t(
-                      'Enables the web notifications feature. if turned off you will be automatically unsubscribed and the notification bell will not be visible',
-                    )}
-                    placement="top"
-                    ml="4px"
+              {featureEnabled ? (
```

Comment:
Global settings should always render no matter user is within the ab testing group or not. Our ab testing only decide whether the switch is default on or off